src/CaseHeader/CaseHeader.tsx

```diff
@@ -31,9 +33,8 @@ const styles = theme => ({
 	},
 });
 
-@translate(['cases'], { wait: true })
-class CaseHeader extends Component {
-	render() {
+class CaseHeader extends React.Component<CaseHeaderProps> {
```

Comment:
IIRC `CaseHeaderProps` is now the generic type for `Component`. But I don't understand the use, can you explain it? This is just because I haven't used Generics since Java a looong time ago :)

Reply:
Well, when using TypeScript, you can use interfaces for describing the types of the `props` and the `state` (`Component<P, S>`). Because for speed I made everything `any` and thats why `CaseHeaderProps` is now a `type`. But it would be better if it was like
```
interface ICaseHeaderProps {
  props here...
}

interface ICaseHeaderState {
  state here...
}
React.Component<ICaseHeaderProps, ICaseHeaderState>
```